rules/falco_rules.yaml

```diff
@@ -3142,6 +3142,17 @@
   priority: CRITICAL
   tags: [process, mitre_privilege_escalation]
 
+# This rule helps detect CVE-2022-0492:
+# A privilege escalation container escaping in cgroup
+- rule: Linux Cgroup Container Escape Vulnerability (CVE-2022-0492)
+  desc: "Detect an attempt to exploit a container escape vulnerability in the Linux Kernel (CVE-2022-0492). By running a container with certains capabilities, a privileged user can modify release_agent file and escape from the container"
+  condition:
+    open_write and fd.name endswith release_agent and (user.uid=0 or thread.cap_permitted contains CAP_DAC_OVERRIDE) and excessively_capable_container
```

Comment:
is this `excessively_capable_container` macro defined somewhere?

Reply:
the macro is introduced in this PR https://github.com/falcosecurity/falco/pull/1963/files done by @loresuso

Reply:
Then what non-default capabilities are needed for the exploit?